build.gradle

```diff
@@ -11,7 +11,7 @@ java {
 }
 
 group = 'cloud.eppo'
-version = '4.0.1'
```

Comment:
Should we should cut a v4.0.1 release to be consistent? The major version bump was for the change in namespace, so this would be a breaking change.
I have a quick PR that I was going to cut a release for (so I could cut v4.1.0 this afternoon)

Reply:
@typotter we never released a v4.0.0 - why not that instead?

Reply:
hm. the latest code has a bugfix for respecting graceful mode that was present in "v4.0.0", but since we never released that version, it wasn't really a bugfix. So ya, good idea - we could tag the repo at v4.0.0 now and cut a release.